projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/merchandising-carousel.e2e-spec.ts

```diff
@@ -76,16 +75,55 @@ function testPDPPage(productId: string): void {
   );
 }
 
-describe.skip('Merchandsing Carousel', () => {
+describe('Merchandsing Carousel', () => {
   beforeEach(() => {
     cy.server();
     cy.route(
       'GET',
       '/strategy/*/strategies/*/products**',
       merchandisingCarousel.STRATEGY_RESPONSE
     ).as('strategyProductsApiRequest');
+    cy.cxConfig({
+      cds: {
+        profileTag: {
+          allowInsecureCookies: true,
+        },
+      },
+    });
+  });
+
+  it('should send another request to the strategy service, if consent is accepted, containing the consent-reference as a HTTP header', () => {
+    testHomePage();
+
+    cy.get('@strategyProductsApiRequest').then(request => {
+      expect(request).to.not.have.property('requestHeaders.consent-reference');
+    });
+
+    merchandisingCarousel.acceptConsent();
+
+    merchandisingCarousel.verifyRequestToStrategyService(strategyRequestAlias, {
+      containsConsentReference: true,
+    });
+
+    merchandisingCarousel.navigateToCategory(camcordersCategoryName);
+    merchandisingCarousel.verifyRequestToStrategyService(strategyRequestAlias, {
+      category: camcordersCategoryCode,
+      containsConsentReference: true,
+    });
   });
 
+  it('should not send another request to the strategy service if consent is not accepted', () => {
+    testHomePage();
+
+    cy.get('@strategyProductsApiRequest').then(request => {
+      expect(request).to.not.have.property('requestHeaders.consent-reference');
```

Comment:
is it necessary to have this in this assertion

Reply:
my argument would be other assertions are already doing verifyRequestToStrategyService (default false for consents) so it checks if it is there or not

Reply:
You're correct, it also means that the second test I'd added is now redundant, so I've removed it